packages/account/src/Sections/Profile/PhoneVerification/phone-verification.scss

```diff
@@ -57,9 +57,14 @@
             height: 40rem;
         }
 
-        .dc-input {
+        &--inputfield {
             width: 60%;
             margin-top: 2.4rem;
+            margin-bottom: 4.4rem;
+            @include mobile {
```

Comment:
We will need to update the css to mobile-screen / whichever suited once tablet design breakpoints in master 🙏

Reply:
Yeappp will update it once tablet design is here 👌